test/compilable/test19746.sh

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env bash
```

Comment:
This is an ill-formed pseudo she-bang header. It only works because the scripts aren't executed, but sourced.
I recommend removing it.

Reply:
Hello @wilzbach, thanks for your feedback.

I am willing to implement your suggestions, but I see that the PR is already merged. Should I still go ahead push the modifications?

Reply:
Yes, it was merged too eagerly. Please do so by opening a new PR. Thanks!